tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/TaskParamsImpl.java

```diff
@@ -847,7 +884,7 @@ public String propagationPolicy() {
         }
     }
 
-    private static class SyncParamsImpl extends TaskParamsImpl implements SyncParams {
+    public static class SyncParamsImpl extends TaskParamsImpl implements SyncParams {
```

Comment:
Is this change to `public` on purpose? It's better to keep these implementations private so if/when we need to refactor we can just do it without worrying about public dependents.